sphinx/ext/imgmath.py

```diff
@@ -197,8 +197,17 @@ def convert_dvi_to_svg(dvipath, builder):
     command.extend(builder.config.imgmath_dvisvgm_args)
     command.append(dvipath)
 
-    convert_dvi_to_image(command, name)
-    return filename, None
+    stdout, stderr = convert_dvi_to_image(command, name)
+
+    depth = None
+    if builder.config.imgmath_use_preview:
+        for line in stderr.splitlines():  # not stdout !
+            matched = depthsvg_re.match(line)
+            if matched:
+                depth = round(float(matched.group(1)) * 100 / 72.27)  # assume 100ppi
```

Comment:
as shown above the depth may have fractional digit. The unit is the TeX point (72.27pt=1in). We could convert it to HTML "pt" unit (72pt=1in) but due to shared code with "png" format, we have to return an integral number of pixels. Thus I round it to an integer here and assume 100ppi.

Clearly any advice welcome on how to better handle this. Probably I have to change more of imgmath to handle a non integer depth. Should we pass it over to html output using pt unit or if we use px how to best convert? The `dvipng` returns an integral number of `px` but I don't know how it does the conversion internally from TeX dimensions to `px` unit.

Reply:
+1: enough to me.

Reply:
I think in future more separate handling of `dvipng` and `dvisvgm` could be considered. Because here I coerce to an `int` mainly due to shared code between the two.

Also, `dvisvgm` can generate its own hashing for naming svg files.